<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Megerősítés</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: rgb(70, 70, 100);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-color: rgb(70, 70, 100);
    font-size: 16px;
    font-weight: bold;
}</string>
    </property>
    <widget class="QPushButton" name="yesButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>128</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Igen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="noButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>128</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Nem</string>
     </property>
    </widget>
    <widget class="QLabel" name="questionLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	font-size: 16pt;
	color: whitesmoke;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
